ludwig/schema/llms/base_model.py

```diff
@@ -54,7 +54,9 @@ def validate(model_name: str):
             if os.path.isdir(model_name):
                 return model_name
             try:
-                AutoConfig.from_pretrained(model_name)
+                # TODO(Arnav): Why is this called twice, and why is trust_remote_code false the second time?
+                # breakpoint()
+                AutoConfig.from_pretrained(pretrained_model_name_or_path=model_name, trust_remote_code=True)
```

Comment:
See if there's a potentially sensible/clear CTA to raise to users when trying to use models that require `trust_remote_code` to be set.

Reply:
Its hard to tell exactly when it happens but i think it may be something to do with the interaction between retrying and the fact that kwargs are popped out rather often:

![image](https://github.com/ludwig-ai/ludwig/assets/15312630/278359e0-052a-4734-8dbf-e41ee9b9957f)